update/delta/delta_1.3.2_001_missing_spatial_indexes.sql

```diff
@@ -0,0 +1,9 @@
+-- Missing spatial indexes
```

Comment:
Should we agree on the delta file name scheme before actually using it? @haubourg any opinion about this?